requirements.txt

```diff
@@ -1,4 +1,4 @@
 llama-stack==0.2.12
 llama-stack-client==0.2.12
 fire==0.7.0
-lightspeed_stack_providers==0.1.8
+lightspeed_stack_providers==0.1.9
```

Comment:
a few Qs here:
- can we just use `>=` instead, and so no need for always changing this version? It has risks, but during development may be useful
- do we stilll need `llama-stack` related libraries, in case we're no longer building the stack image ourselves?

Reply:
Yes, I can loosen the versions :+1: 

I have tried removing the `llama-stack` dependencies (you are correct, we should not need them now).

Unfortunately `lightspeed-stack` is failing to start (an [issue](https://redhat-internal.slack.com/archives/C07MC7G9T8A/p1750862662419639) _their_ side).

When I confirm our container runs OK I'll remove the dependencies.